docs/content/community/contributing.mdx

```diff
@@ -147,19 +147,28 @@ To run the Dagster documentation website locally, run the following commands:
 
 ```bash
 cd docs
-yarn --cwd next dev  # Serves the docs website on http://localhost:3001
+next-watch-build   # Serves the docs website on http://localhost:3001
 ```
 
-Troubleshooting tip: You may need to run `yarn --cwd next` (without the `dev` command) first to install dependencies. Also make sure that your Node version is >=12.13.0.
+Troubleshooting tip: You may need to run `make next-dev-install` first to install dependencies. Also make sure that your Node version is >=12.13.0.
 
-The API documentation is generated from ReStructured Text files (`.rst`), which extracts Python docstrings from the library files. The `.rst` files can be found in the `docs/sections/api/apidocs` directory.
+The API documentation is generated from ReStructured Text files (`.rst`), which extracts Python docstrings from the library files. The `.rst` files can be found in the `docs/sphinx/sections/api/apidocs` directory.
 
 If you change any `.rst` files, be sure to run the following command in the `docs` directory:
 
 ```bash
-make build
+make apidoc-build
 ```
 
+The majority of our code snippets are pulled from real python files. This allows us to test our code snippets and ensure they remain up-to-date. In `.mdx` files you'll see a code snippet with `python file=/path/to/file.py startafter=start_marker endbefore=end_marker` at the beginning of the block. You can find the corresponding python file at `dagster/examples/docs_snippets/docs/snippets/path/to/file.py`. The code included in each snippet is the code in the file between the `# start_marker` and `# end_marker` comments. If you change a code snippet, run:
+
+```bash
+make mdx-format
+```
+from the `docs` directory.
```

Comment:
```suggestion
The majority of our code snippets are pulled from real Python files. This allows us to test our code snippets and ensure they remain up-to-date. 

In `.mdx` files, you'll see a code snippet with `python file=/path/to/file.py startafter=start_marker endbefore=end_marker` at the beginning of the block. For example:

[add example here]

You can find the corresponding Python file at `dagster/examples/docs_snippets/docs/snippets/path/to/file.py`. The code included in each snippet is the code in the file between the `# start_marker` and `# end_marker` comments. 

If you change a code snippet, run the following from the `docs` directory:

```bash
make mdx-format
```
```

Capitalization and some formatting. The paragraph was a bit dense so I broke it up to make it easier to skim.

Consider adding an example of the code snippets here to demonstrate what's being discussed, too.